canonicalwebteam/blog/wordpress_api.py

```diff
@@ -43,11 +43,7 @@ def build_url(endpoint, params={}):
 
     query = urlencode(clean_params)
```

Comment:
What do you thing about making this line `urlencode({**clean_params, '_embed': "true"})`, and then `return f"{API_URL}/{endpoint}?{query} `

Reply:
Nice one. Updated it. As discussed and test going from `&_embed` to `&_embed=true` does not seem to influence the functionality